__tests__/app.test.js

```diff
@@ -126,6 +126,42 @@ describe("GET /api/articles", () => {
   });
 });
 
+describe("GET /api/articles - topic query", () => {
```

Comment:
You need another test to check we get an empty array back when there are no articles for a given topic that does exist.